nncf/common/accuracy_aware_training/training_loop.py

```diff
@@ -24,20 +24,36 @@
 from nncf.common.utils.logger import logger as nncf_logger
 from nncf.common.utils.registry import Registry
 from nncf.config.config import NNCFConfig
-from nncf.common.utils.backend import __nncf_backend__
-
-if __nncf_backend__ == 'Torch':
-    from nncf.torch.accuracy_aware_training.runner import PTAccuracyAwareTrainingRunner as \
-        AccuracyAwareTrainingRunner
-elif __nncf_backend__ == 'TensorFlow':
-    from nncf.tensorflow.accuracy_aware_training.runner import TFAccuracyAwareTrainingRunner as \
-        AccuracyAwareTrainingRunner
 
 
 ModelType = TypeVar('ModelType')
 ADAPTIVE_COMPRESSION_CONTROLLERS = Registry('adaptive_compression_controllers')
 
 
+def get_backend_specific_training_runner_cls(compression_controller: CompressionAlgorithmController,
```

Comment:
Please, move this function to a separate module as you suggested.

Reply:
Done